contrib/pg_tde/documentation/docs/setup.md

```diff
@@ -1,12 +1,16 @@
 # Set up `pg_tde`
 
+The steps below describe the default key configuration. This means that the entire PostgreSQL cluster is encrypted using the same global encryption key. For how to configure multi-tenancy, see the [Set up multi-tenancy](multi-tenant-setup.md) guidelines.
```

Comment:
This isn't technically true. Using a default key does not necessarily mean that the whole cluster is encrypted with the same key. It is a good way to do so but for example as soon as you start rotating keys this will no longer be the case.